Features/WalletTab/Sources/Scenes/WalletScene.swift

```diff
@@ -65,6 +65,15 @@ public struct WalletScene: View {
                     showBalancePrivacy: $preferences.isHideBalanceEnabled
                 )
                 .listRowInsets(.assetListRowInsets)
+            } header: {
+                if model.isLoadingAssets {
+                    HStack {
+                        Text("Syncing wallet…")
```

Comment:
```suggestion
                        Text(Localized.Common.loading + "...")
```

for now let's use this